New Security Ship; the SEC-Sunrise, a Ship Made for Detectives and Lonesome Officers

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
I added the SEC-Sunrise ship, a detective style ship for Station Guard players, since they really only had the Arachnid. 

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
This ship, while lightly armed and armored and would most likely die in a fight, is equipped with a Radar console, as well as multiple Security grade thrusters, allowing them to run away from any fight they can't take. 

Whilst lightly armed and armored, they do come equipped with EMP launchers, so while they can theoretically subdue any small time smugglers, or CIV class ships, they would be better either acting as support, or running away from the fight at full speed. 

## How to test
<!-- Describe the way it can be tested -->
Run the .yml file with loadgrid, should work. 


## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

<img width="977" height="792" alt="newsunrise live" src="https://github.com/user-attachments/assets/2a4d32da-a54d-432e-8a2e-78e0e219c069" />
<img width="992" height="790" alt="newsunrise" src="https://github.com/user-attachments/assets/22ccb563-154c-469f-8b22-9b07a6098740" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: SEC-Sunrise

Rubeebeebee committed:
I want to add another gas mining ship- so here is the HES-Phaeron, a tanker ship with a huge tank, but you'll have to carry either one filtered gas, or unfiltered gas.

Rubeebeebee committed:
Revert "I want to add another gas mining ship- so here is the HES-Phaeron, a tanker ship with a huge tank, but you'll have to carry either one filtered gas, or unfiltered gas."

This reverts commit c4717d01d2f34a267d2ef4055c488270bb393b21.

Rubeebeebee committed:
A new ship for SEC players, The Sunrise; a detective ship

monolith8319 committed:
i added more decals

chore: automatically update REUSE headers

monolith8319 committed:
chore: automatically update REUSE headers

Redrover1760 commented on Resources/Maps/_Mono/Shuttles/sunrise.yml:
```diff
@@ -0,0 +1,3398 @@
+      locked: False
+    - type: BecomesStation
+      id: Sunrise
+- proto: ActionToggleLight
```
this definitely shouldn't be here, mapinit'd ``- proto: ActionToggleLight``

I recommend trying to delete it via text editor after saving a backup and making sure the map doesn't break.

Rubeebeebee replied:
thanks! ill do this right now

Rubeebeebee replied:
> proto: ActionToggleLight

i deleted this line and it uh... broke. i can't buy it anymore in the shipyard after testing.

Rubeebeebee replied:
> this definitely shouldn't be here, mapinit'd `- proto: ActionToggleLight`
> 
> I recommend trying to delete it via text editor after saving a backup and making sure the map doesn't break.

no wait nevermind, i copied by the backup back and tried deleting it again, worked this time around!

Rubeebeebee replied:
ive commited the change, thank you!

Rubeebeebee committed:
Merge branch 'Sunrise-Branch' of https://github.com/Rubeebeebee/Monolith into Sunrise-Branch

Rubeebeebee committed:
deleted - proto: ActionToggleLight thanks to Redrover1760's suggestion. idk why

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Sunrise-Branch

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/Sr/sunrise.yml:
```diff
@@ -0,0 +1,37 @@
+- type: vessel
+  id: Sunrise
+  parent: BaseVessel
+  name: Sunrise
```
you can add a company tag here if you want

Rubeebeebee replied:
oh yeah, ill add RSS for RedStar Security

Rubeebeebee replied:
changed it!

Rubeebeebee replied:
> you can add a company tag here if you want

apparently adding the RSS for the company tag breaks the .yml, dunno why, removed it though so it works now.

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/Sr/sunrise.yml:
```diff
@@ -0,0 +1,37 @@
+  id: Sunrise
+  parent: BaseVessel
+  name: Sunrise
+  description: A Detective's ship. Comes equiped with a suite of forensic tools for investigative work. Capable of subduing smugglers and CIV ships with it's EMP launchers, and running away from anything more dangerous. 
```
maybe private investigator's ship? Or could just be Detective works fine

Rubeebeebee replied:
ill make it so it says private investigator

Rubeebeebee replied:
changed it too!

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/Sr/sunrise.yml:
```diff
@@ -0,0 +1,37 @@
+  mapPath: /Maps/_Mono/Shuttles/sunrise.yml
+  minPlayers: 0
+  stations:
+    Arachnid:
```
Becomesstation ID not replaced

Rubeebeebee replied:
ope, thats a boneheaded mistake on my part! ill replace it ASAP

Rubeebeebee replied:
changed it to Sunrise, hopefully it works this time.

Redrover1760 reviewed (changes requested):
Fix map schema error

Redrover1760 commented on Resources/Maps/_Mono/Shuttles/sunrise.yml:
```diff
@@ -0,0 +1,3537 @@
+-
+  entities:
+  - uid: 8
+    mapInit: true
```
initialized/paused shouldn't be here, this entity probably needs to get removed entirely via text editor. Make a copy before doing so, can break the map.

Rubeebeebee replied:
> initialized/paused shouldn't be here, this entity probably needs to get removed entirely via text editor. Make a copy before doing so, can break the map.

gotcha, should i remove these three lines entirely?

Rubeebeebee replied:
> initialized/paused shouldn't be here, this entity probably needs to get removed entirely via text editor. Make a copy before doing so, can break the map.

there we go! did what you asked, and i tested it to make sure its working via buying it at the shipyard 
<img width="987" height="568" alt="mapinit was key" src="https://github.com/user-attachments/assets/185385be-833d-4ae8-81eb-c40f462f8808" />

Rubeebeebee replied:
removed the desk lamp, apparently that was causing the mapinit issues. 
<img width="775" height="738" alt="swag" src="https://github.com/user-attachments/assets/861dd5be-de47-491b-800a-191a3407f151" />

Rubeebeebee committed:
added a company tag, changed the line "A Detective's ship..." Into "A private investigator's ship..." and changed the becomestationid from Arachnid to Sunrise (thanks @redrover1760 !)

Rubeebeebee committed:
removed mapInit: true in the .yml code for the map

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
removed the desk lamp because its causing mapinit issues. Added a whiskey bottle + a shotglass to replace it.

Also added trim to the carpet on the dock area, plus a box of forensic pads for any investigating

monolith8319 committed:
chore: automatically update REUSE headers

Redrover1760 reviewed (changes requested):
initialized appraisegrid states 55k, should cost more than 55k credits

remove the NF detective PDA, it comes with an ID with TSFMC access.

Rubeebeebee commented:
> initialized appraisegrid states 55k, should cost more than 55k credits
> 
> remove the NF detective PDA, it comes with an ID with TSFMC access.

yessir, will do!

Rubeebeebee committed:
adjusted price to 59,000 spessos, removed detective PDA

Rubeebeebee commented:
> initialized appraisegrid states 55k, should cost more than 55k credits
> 
> remove the NF detective PDA, it comes with an ID with TSFMC access.

Done! removed the PDA, and adjust the price to 59,000 instead of of the original 49,000

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Sunrise-Branch

Rubeebeebee commented:
synched the branch! should be updated now!

Rubeebeebee committed:
replaced the deckard revolver with the Inspector, beause the deckard had 5 shots, the inspector has 6, which is more in line with the speedloaders found in the gunsafe.

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
updated the shipyard .yml to actually work now! hooray!
